R/PostDefinition.R

```diff
@@ -88,7 +110,13 @@ postDefinition <- function(baseUrl, name, category, definition) {
   response <- httr::content(response)
   structureCreated <- response
   response$expression <- NULL
-
+  
+  if(category %in% c("pathway")){
```

Comment:
I think we can support more categories using this framework

Reply:
Good to hear. I am happy to take a look at the remaining categories